src/style.css

```
@@ -1,3 +1,79 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  padding: 0 50px;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+@media (max-width: 1023.98px) {
```

Don`t copy fractional numbers from figma. They likely not work anyway. 
```suggestion
@media (max-width: 1024px) {
```

---

It's not from figma. Max-width: 1023.98px media work from 1024px screen, and max-width: 1024px will work only from 1025px screen. As for me, float number is better value for @media, but there's no problem, for me, to chenge in this task

---

I decided to delete @media. Because in test @media (max-width: 1024px) created diff with "Header_tag_1024px.png". Anyway this header look like only for desktop